testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -165,21 +165,21 @@ def outfile(self, tmpdir):
 
     @pytest.fixture()
     def correct_values(self):
-        return [[0, 1000, 0], [49, 1049, 4.68953]]
+        return [[0, 1, 0], [49, 50, 4.68953]]
```

Comment:
@orbeckst this makes the tests pass. I'm unsure why we have to adjust the step by 1 position now.

Reply:
Do you mean, adjusting the step by 1 position instead of 1000? The reason for this is that `istart` is measured in integrator time steps. 1 frame contains `nsavc` integrator time steps. If a DCD wants to state that the trajectory starts "at frame 1" then it has to set `istart = nsavc`.  In 0.17.0 we wrongly interpreted `istart` as frames instead of integrator time steps.

If your question is "why do our test DCDs now have a starting time "at frame 1", e.g.
```python
>>> import MDAnalysis as mda
... from MDAnalysisTests.datafiles import PSF, DCD, DCD_NAMD_GBIS, PSF_NAMD_GBIS, TPR, XTC
...
>>> # DCD time starts at 1*dt
>>> u = mda.Universe(PSF, DCD)
>>> print(u.trajectory.frame, u.trajectory.time, u.trajectory.n_frames, u.trajectory.totaltime)
0 0.9999999119200186 98 96.9999914562418
>>> u = mda.Universe(PSF_NAMD_GBIS, DCD_NAMD_GBIS)
>>> print(u.trajectory.frame, u.trajectory.time, u.trajectory.n_frames, u.trajectory.totaltime)
0 0.010000000029814058 100 0.9900000029515917
>>> # XTC starts at 0*dt
>>> u = mda.Universe(TPR, XTC)
>>> print(u.trajectory.frame, u.trajectory.time, u.trajectory.n_frames, u.trajectory.totaltime)
0 0.0 10 900.0000686645508
```

then the answer is because this is how we interpret the meaning of `istart` at the moment: as an offset of `istart/nsavc` frames from 0 (based on the docs that I cited in  #1819). I am admittedly not 100% sure if this is the best way to do this; it seems to imply that the starting frame is *not* saved in a trajectory and the first frame that was saved would be the one `istart` integrator time steps after the begin of the simulation. The CHARMM docs (Example under [TRAJECTORY](https://www.charmm.org/charmm/documentation/by-version/c42b1/params/doc/dynamc/#Trajectory) command) imply that the [DYNAmics](https://www.charmm.org/charmm/documentation/by-version/c42b1/params/doc/dynamc/#Syntax) command does indeed start saving *after* the coordinates with the initial conditions:

> Example. Assume that the three files traj1.trj, traj2.trj and traj3.trj were created using the following dyanmics commands: 
>         dyna start nstep 1000 nsavc 100 ! saves 10 frames (100, 200, ...1000) 
>         dyna restart nstep 10000 nsavc 100 ! saves 100 frames (1100, 1200, ...10100) 
>         dyna restart nstep 10000 nsavc 100 ! saves 100 frames (10200, 10300, ...20100) 

I might have to play around with the CHARMM [TRAJectory](https://www.charmm.org/charmm/documentation/by-version/c42b1/params/doc/dynamc/#Trajectory) command to get a better idea of what CHARMM really thinks of a trajectory. 

However, I'd like to fix `istart` first (which is clearly wrong) and then discuss what is more of a discussion about how to interpret quirks in the DCD format and how to make it work seamlessly in MDAnalysis.

Reply:
T1000 to 1 is OK. We made this change in the current development cycle.  I'm curious why we needed to change from step 49 to 50.  My first try to fix the istart behavior was in #1767. The change in the iteration introduced by your changes is new to this pr. Because such a change would break user code I'm hesitant to merge this.   

There are some NAMD experts in my group. I will ask them what behavior is best for them.

Reply:
Can you explain with a simple example what the change "from 49 to 50" means?

Reply:
I quickly browsed through the discussion in #1767. The choice istart=1 is now istart=None or manually setting istart=nsavc. Setting istart=0 as the default for the writer is retained. 

If we wanted to make clearer that we want the user to be able to set *frame* offsets then we should introduce something like frame_offset. We already have offset for the Timestep. 

I haven't understood yet where changes in frames come from. It might well be that something needs to be fixed but I'd like to have a test case for this in the dcd tests (and not primarily in the rms code).

Reply:
> I'm curious why we needed to change from step 49 to 50.

I think the same reason: before we had 1049, which was calculated as istart + frames = 1000 + 49. Now we have istart/nsavc + frames = 1000/1000 + 49 = 1 + 49 = 50.

Reply:
Sorry I didn't have time yet to look at this. But your guess is likely right. This means from now on all reported time will start from 1? except when istart is 0.

Reply:
I'm cool with that. This will affect users by showing later times now. I would change the CHANGELOG message to say that we used to report to early timestamps and now read times correctly and users should expect that times are shifted to later values. I like to be a bit more exact then just *we did it wrong*

Otherwise looks good to me. It can be merged.